backend.dev.env.example

```diff
@@ -0,0 +1,2 @@
+# Token from mapbox.com to get Open Street Map tiles
+MAPBOX_TOKEN=
```

Comment:
No `SECRET_KEY`?

Reply:
The dev secret key was hardcoded. I created this file with the miminum necessary contents to successfully run a local development docker. This was broken with the introduction of MAPBOX_TOKEN.